cloudslang-entities/src/main/java/io/cloudslang/lang/entities/bindings/values/SensitiveValue.java

```diff
@@ -29,6 +31,7 @@
 public class SensitiveValue implements Value {
 
     public static final String SENSITIVE_VALUE_MASK = "********";
+    public static final String ENCRYPTOR_PACKAGE = "com.hp.onyx.commons.encryption";
```

Comment:
I still don't think we should put company related info in open source and make CS dependent on OO codebase - I would rather just pass a single constructor param to set the mode -- encryption / obfuscation or maybe a System Property -- that way we could easily add unit tests at CS level -- if somebody wants to modify the obfuscation / enc behavior he can do it anyway

@lucian-cm WDYT?